src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(x) {
+      return calculator.result + x;
+    },
```

Comment:
Let's rewrite your solution for more practice with `this` in JS.
Add `x` is not clear name. Use `value` or `number` insteat
```suggestion
      return this.result + x;
    },
```